### PR TITLE
EQL: Plug query params into the AstBuilder

### PR DESCRIPTION
**Pull request by costin**
As the eventType is customizable, plug that into the parser based on the
given request.

### CODE REVIEW

**Comment by costin** [comment]
Only the last commit count - the PR is raised against https://github.com/elastic/elasticsearch/pull/51872 (which hasn't been merged yet in master).

File: x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/RequestDefaults.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ */
+
+package org.elasticsearch.xpack.eql.action;
+
+public final class RequestDefaults {
+
+    private RequestDefaults() {}
+
+    public static final String FIELD_TIMESTAMP = "@timestamp";
+    public static final String FIELD_EVENT_TYPE = "event_type";
```
Comment (rw-access) [inline x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/RequestDefaults.java]: I think `event.category` is the best fit, but we can come back to this in another issue/PR.

**Review by rw-access — APPROVED** [review]

**Review by aleksmaus — APPROVED** [review]

**Review by astefan — APPROVED** [review]
LGTM

**Comment by costin** [comment]
@elasticmachine merge upstream

**Pull request merged**